tensorflow/python/training/adam.py

```diff
@@ -29,7 +29,7 @@
 
 
 class AdamOptimizer(optimizer.Optimizer):
-  """Optimizer that implements the Adam algorithm.
+  """Optimizer that implements the Adam algorithm (http://arxiv.org/abs/1412.6980).
```

Comment:
I think what you have looks good. Can you remove that other reference and maybe point to specific version of the paper (v7)? The formulas were updated from previous versions.